pkg/api/v1beta1/awspcaissuer_types.go

```diff
@@ -36,7 +35,28 @@ type AWSPCAIssuerSpec struct {
 	Region string `json:"region,omitempty"`
 	// Needs to be specified if you want to authorize with AWS using an access and secret key
 	// +optional
-	SecretRef v1.SecretReference `json:"secretRef,omitempty"`
+	SecretRef SecretReference `json:"secretRef,omitempty"`
+}
+
+type SecretReference struct {
+	// Name is unique within a namespace to reference a secret resource.
+	// +optional
+	Name string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	// Namespace defines the space within which the secret name must be unique.
+	// +optional
+	Namespace string `json:"namespace,omitempty" protobuf:"bytes,2,opt,name=namespace"`
+	// Specifies the secret key where the AWS Access Key ID exists
+	// +optional
+	AccessKeyIDSelector SecretSelector `json:"accessKeyIDSelector,omitempty"`
+	// Specifies the secret key where the AWS Secret Access Key exists
+	// +optional
+	SecretAccessKeySelector SecretSelector `json:"secretAccessKeySelector,omitempty"`
+}
+
+// SecretSelector references a key within a Secret resource.
+type SecretSelector struct {
+	// The key of the entry in the Secret resource's `data` field to be used.
+	Key string `json:"key,omitempty"`
```

Comment:
No need to define this yourself. There already exists a type in the K8s core library for `SecretKeySelector` https://pkg.go.dev/k8s.io/api/core/v1#SecretKeySelector